sros/Makefile

```diff
@@ -13,3 +13,10 @@ VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9]\([0-9]\+\.[0-9]\+\.[A-Z][0-9
 
 -include ../makefile-sanity.include
 -include ../makefile.include
+
+# We (TeraStream) do not have a license file for 21.10.R2, so it seems like we
+# cannot run it?!
+ifeq ($(VERSION),21.10.R2)
```

Comment:
Just remove the image from the secret volume instead of skipping it like this?

Reply:
Hehe, yes, just need to remember to not copy it again :)